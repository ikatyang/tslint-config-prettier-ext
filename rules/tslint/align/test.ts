export function test(a: number,
  b: number) {}
