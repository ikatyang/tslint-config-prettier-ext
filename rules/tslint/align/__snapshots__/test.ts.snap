// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should be pretty after formatting 1`] = `
"
<<<<<<< before
export function test(a: number,
  b: number) {}

=======
export function test(a: number, b: number) {}

>>>>>>> after
"
`;

exports[`should fail before formatting 1`] = `
"ERROR: (align) /rules/tslint/align/test.ts[2, 3]: parameters are not aligned
"
`;
